graylog2-web-interface/src/components/navigation/Navigation.jsx

```diff
@@ -90,7 +90,7 @@ const Navigation = ({ permissions, fullName, location, loginName }) => {
 
         {
         AppConfig.gl2DevMode()
-          && <Badge bsStyle="danger" className="small-scrn-badge dev-badge">DEV</Badge>
+          && <Badge bsStyle="danger" className="small-scrn-badge dev-badge">{AppConfig.isCloud() ? 'CLOUD ' : ''} DEV</Badge>
```

Comment:
I'd remove the prefix space here

There is already a space before `DEV` so I think we can leave it out of the `'CLOUD '` string, or remove the space before `DEV`
```suggestion
          && <Badge bsStyle="danger" className="small-scrn-badge dev-badge">{AppConfig.isCloud() ? 'CLOUD' : ''} DEV</Badge>
```